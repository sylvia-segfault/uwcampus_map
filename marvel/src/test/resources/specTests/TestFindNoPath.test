# test the edge case when there is no path between two
# heroes

LoadGraph graph isolatedNode.tsv
FindPath graph A G