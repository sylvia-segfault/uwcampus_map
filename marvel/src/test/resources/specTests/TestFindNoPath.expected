# test the edge case when there is no path between two
# heroes

loaded graph graph
path from A to G:
no path found